src/utils/logger.js

```diff
@@ -0,0 +1,14 @@
+const pino = require('pino');
+
+const logger = pino({
+  transport: {
+    target: 'pino-pretty',
+    options: {
+      colorize: true,
+      translateTime: 'SYS:dd-mm-yyyy HH:MM:ss',
+      ignore: 'pid,hostname',
+    },
+  },
+});
```

Comment:
Can you configure the logger to save logs to a log file? This is important once the project is deployed, then errors can be saved and checked from the log file. But you have to ensure that the logs show in the console for local development. [This article](https://betterstack.com/community/guides/logging/node-js/how-to-install-setup-and-use-pino-to-log-node-js-applications) has some good examples.

Reply:
add logger destination